.github/workflows/docs.yml

```
@@ -22,7 +22,7 @@ on:
 
 jobs:
   publish:
-    uses: Alfresco/jekyll-build-tools/.github/workflows/jekyll-publish.yml@5ee00079dbc774857cafa61054385d98a6f52374
+    uses: Alfresco/jekyll-build-tools/.github/workflows/jekyll-publish.yml@3552fc7b84bebc88545e7d78e64e2ff85388877a
```

```suggestion
    uses: Alfresco/jekyll-build-tools/.github/workflows/jekyll-publish.yml@3552fc7b84bebc88545e7d78e64e2ff85388877a # v0.1.1
```

just created such tag